INSS.EIIR.Web/Controllers/SearchController.cs

```diff
@@ -29,6 +29,10 @@ public IActionResult Search(string searchTerm)
             {
                 return RedirectToAction("Index", new { error = true });
             }
+            if (searchTerm.Contains('\t'))
```

Comment:
whats the ticket number in our Jira board for this change

can you also add test coverage for tabs with urlencoding